src/vcState.h

```diff
@@ -115,10 +122,12 @@ struct vcState
   bool forceLogout;
   double lastServerAttempt;
   double lastServerResponse;
-  vdkContext *pVDKContext;
-  vcRenderContext *pRenderContext;
+  vcContextIndex eCurrentContextIndex;
+  vdkContext *pVDKContext[vcContext_Count];
+  vcRenderContext *pRenderContext[vcContext_Count];
```

Comment:
There doesn't need to be multiple render contexts either, the internals can be shared without issue

Reply:
I'm making a few changes to support multiple scenarios using one render context.

Reply:
I'm making a few changes to support multiple scene using one render context.